src/GTP.cpp

```diff
@@ -161,6 +163,11 @@ std::string GTP::get_life_list(const GameState & game, bool live) {
 
 bool GTP::execute(GameState & game, std::string xinput) {
     std::string input;
+    // This is a way to make sure e.g. cfg_max_playouts is
+    // initialized before we construct the search object.
```

Comment:
I think that logic can go to the top of think(). (Or if you follow the previous suggestion, the problem disappears)

Reply:
fixed